pkg/client/gen.go

```diff
@@ -81,43 +74,32 @@ func (*SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 	}
 
 	plugins := []*manifest.Manifest{}
-	if includes(cfg.Config.PluginSelections, e2ePluginName) {
-		plugins = append(plugins, e2eManifest(cfg))
-	}
-	if includes(cfg.Config.PluginSelections, systemdLogsName) {
-		plugins = append(plugins, systemdLogsManifest(cfg))
+	for _, v := range cfg.DynamicPlugins {
+		switch v {
+		case e2ePluginName:
+			plugins = append(plugins, e2eManifest(cfg))
+		case systemdLogsName:
+			plugins = append(plugins, systemdLogsManifest(cfg))
+		}
 	}
 
 	pluginYAML := []string{}
-	for _, v := range plugins {
+	for _, v := range append(plugins, cfg.StaticPlugins...) {
 		yaml, err := kuberuntime.Encode(manifest.Encoder, v)
 		if err != nil {
 			return nil, errors.Wrapf(err, "serializing plugin %v as YAML", v.SonobuoyConfig.PluginName)
 		}
 		pluginYAML = append(pluginYAML, strings.TrimSpace(string(yaml)))
 	}
 
-	// Template values that are regexps (`E2EFocus` and `E2ESkip`) are
-	// embedded in YAML files using single quotes to remove the need to
-	// escape characters e.g. `\` as they would be if using double quotes.
-	// As these strings are regexps, it is expected that they will contain,
-	// among other characters, backslashes. Only single quotes need to be
-	// escaped in single quote YAML strings, hence the substitions below.
-	// See http://www.yaml.org/spec/1.2/spec.html#id2788097 for more details
-	// on YAML escaping.
```

Comment:
All this is now handled by the yaml library which serializes it.  Should probably try to validate cases like this though; I'm not 100% sure there was sufficient testing here. The proof isnt that _our_ tests pass, but that the e2e tests would select properly. We may have been too opinionated in certain cases and the yaml serializer knows best.